tests/index.test.js

```diff
@@ -191,3 +221,38 @@ describe('toggleCreator', () => {
     });
   });
 });
+
+describe('handleSumbit', () => {
+  describe('with React Component', () => {
+
+    let wrapper;
+    let submit;
+
+    beforeEach(() => {
+      wrapper = mount(<FormComponent />);
+      submit = expect.spyOn(Actions, 'submit');
+
+      wrapper.find('button').get(0).click();
+    });
+
+    afterEach(() => {
+      expect.restoreSpies();
+    })
+
+    it('returns a value for an input', () => {
+      expect(Actions.submit.calls[0].arguments[0].testInput).toEqual('value');
```

Comment:
Since each of these tests is getting daata from the same place, it could be extracted :)